google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -50,6 +50,7 @@
   <xi:include href="/META-INF/google-cloud-core.xml" xpointer="xpointer(/idea-plugin/*)"/>
   <xi:include href="/META-INF/google-cloud-storage.xml" xpointer="xpointer(/idea-plugin/*)"/>
   <xi:include href="/META-INF/google-cloud-repos.xml" xpointer="xpointer(/idea-plugin/*)"/>
+  <xi:include href="/META-INF/google-cloud-maven.xml" xpointer="xpointer(/idea-plugin/*)"/>
```

Comment:
I think you want to invert this so that this import is optional and the google-cloud-maven.xml uses xi:include.  Once you decouple libraries support from maven you'll probably want to also include a libraries maven plugin fragment into google-cloud-maven.xml but you can only have one optional inclusion per module type.

Reply:
ah yeah. thanks. I initially tried having an optional depends here, and then a regular depends in "google-cloud-maven". but that clearly wont work.